users/service_test.go

```diff
@@ -1101,6 +1103,14 @@ func TestUpdateClientRole(t *testing.T) {
 			updateRoleErr:        svcerr.ErrAuthentication,
 			err:                  svcerr.ErrAuthentication,
 		},
+		{
```

Comment:
Add missing test case. You can run the following command and look at the HTML output for the missing case
```bash
cd users && go test -v --race -covermode=atomic -coverprofile=cover.out . && go tool cover -html cover.out -o cover.html
```